src/gs.c

```diff
@@ -1619,3 +1622,12 @@ void fgs_free(const sint *handle)
   fgs_info[*handle] = 0;
 }
 
+void fgs_unique(slong id[], const sint *n, const MPI_Fint *c, const sint *np)
```

Comment:
Why not `const slong id[]`?

Reply:
The gs_unique routine modifies the id[] array (by changing the sign of some of its elements) so it cannot be constant.

As a side remark: it is possible to perform a unique gs_setup without modifying the id[] array by setting the variable unique=1 but there is currently no Fortran wrapper to do so. If there is any interest in adding one, we can start a discussion in a separate issue.